src/lib/Lower.hs

```diff
@@ -323,6 +324,7 @@ vectorizeLoops vectorByteWidth abs = liftEnvReaderM do
       Failure errs -> error $ pprint errs
       Success (block', errs) -> return $ (Abs d block', errs)
 {-# INLINE vectorizeLoops #-}
+{-# SCC vectorizeLoops #-}
```

Comment:
I think you have to take out `INLINE` if you want the `SCC` to work. But it's a good idea to not inline it anyway, so maybe let's do it.

Reply:
Done.